DataConnectors/CyberArk/CyberArk Data Connector.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id": "CyberArk",
+    "title": "CyberArk Enterprise Password Vault (EPV) Syslog",
```

Comment:
You might want to remove Syslog from the title as it's basically a CEF connector